### PR TITLE
Migrate test Visualization->"One part of decomposed object visible" to 3.0

### PR DESCRIPTION
**Pull request by leosobral**

### CODE REVIEW

**Review by cschladetsch — COMMENTED** [review]
Why is there extra white space around indented bullet point, and why do some bullet points end with punctuation and others not?

File: Test Cases/v3.0/Visualization/One part of decomposed object visible/unzipped/extensions.xsd [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+        <TopicType>Warning</TopicType>
+        <TopicType>Info</TopicType>
+        <TopicType>Unknown</TopicType>
+    </TopicTypes>
+    <TopicStatuses>
+        <TopicStatus>Open</TopicStatus>
+        <TopicStatus>Closed</TopicStatus>
+    </TopicStatuses>
+    <Priorities/>
+    <SnippetTypes/>
```
Comment (jasollien) [inline Test Cases/v3.0/Visualization/One part of decomposed object visible/unzipped/extensions.xsd:14]: Users must be present, even if an empty array

File: Test Cases/v3.0/Visualization/One part of decomposed object visible/unzipped/3146b55f-ac72-4490-bb06-60cc254b3d8e/markup.bcf [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+<Markup xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="markup.xsd">
+    <Header>
+        <Files/>
+    </Header>
+    <Topic Guid="3146b55f-ac72-4490-bb06-60cc254b3d8e" ServerAssignedId="829" TopicType="Error" TopicStatus="Open">
+        <ReferenceLinks/>
+        <Title>One plate visible</Title>
+        <Labels/>
+        <CreationDate>2017-05-23T08:22:35.531Z</CreationDate>
+        <CreationAuthor>pasi.paasiala@solibri.com</CreationAuthor>
```
Comment (pasi-paasiala) [inline Test Cases/v3.0/Visualization/One part of decomposed object visible/unzipped/3146b55f-ac72-4490-bb06-60cc254b3d8e/markup.bcf:11]: could use a generic email here.

**Comment by pasi-paasiala** [comment]
The bcf file has _MACOSX folder.

**Comment by ykulbak** [comment]
Imports correctly on Aconex

File: Test Cases/v3.0/Visualization/One part of decomposed object visible/unzipped/extensions.xml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Extensions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="extensions.xsd">
```
Comment (CBenghi) [inline Test Cases/v3.0/Visualization/One part of decomposed object visible/unzipped/extensions.xml:2]: Is the usage of local schemas ok?

**Review by CBenghi — APPROVED** [review]
I'm generally ok with this test, but I have a question on the correctness of using schema references inside the xml

**Pull request merged**